stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -168,7 +168,9 @@ spec:
             - name: JAVA_OPTS
               value: >
                 {{ default "" .Values.master.javaOpts }}
-                {{- if .Values.master.sidecars.configAutoReload.enabled }} -Dcasc.reload.token=$(POD_NAME) {{end}}
+                {{- if .Values.master.sidecars.configAutoReload.enabled }}
```

Comment:
@kimxogus What's the change except formatting here?

Reply:
If it's about having it on a new line then we could simply change it from `{{- if` to `{{ if`

Reply:
Removed `-`.
There's no change except here